.aicoe-ci.yaml

```diff
@@ -0,0 +1,4 @@
+# Setup and configuring aicoe-ci with configuration file `.aicoe-ci.yaml`
```

Comment:
Can we link to some documentation in here? Like.. list of checks available etc.? :slightly_smiling_face: